pages/validator/external-chains/optimism.md

```diff
@@ -0,0 +1,300 @@
+
+## Requirements
+
+- [Setup your Axelar validator](/validator/setup)
+- Minimum hardware requirements: 8 AWS vCPU+, 16GB RAM, 80GB+ free storage space.
+- MacOS or Ubuntu 20.04 (tested on 20.04)
+- [Official Documentation](https://community.optimism.io/docs/developers/build/run-a-node/#)
+
+
+## Prerequisites
+```bash
+sudo apt-get install jq -y
+snap install docker 
+```
+
+## Clone Repo
+
+```bash
+git clone https://github.com/ethereum-optimism/simple-optimism-node.git
+cd simple-optimism-node
+cp .env.example .env
+```
+
+## Set environment variables
+
+
+<Tabs tabs={[
+{
+title: "Mainnet",
+content: <CodeBlock language="bash">
+{`
+NETWORK_NAME=mainnet
+# can be full or archive
+NODE_TYPE=full
+# only download option is available for now
+BEDROCK_SOURCE=download
+# L1 node that the op-node will get chain data from - please use your own ETH node
+OP_NODE__RPC_ENDPOINT=
+# Use basic for OP_NODE__RPC_TYPE if you are using your own node in OP_NODE__RPC_ENDPOINT
+# Other options are: alchemy, quicknode, infura, parity, nethermind, debug_geth, erigon
+# Depending upon which provider you are using in OP_NODE__RPC_ENDPOINT, mentioning the right
+# OP_NODE__RPC_TYPE will optimize the process
+OP_NODE__RPC_TYPE=
+`}
+</CodeBlock>
+},
+{
+title: "Testnet",
+content: <CodeBlock language="bash">
+{`
+# only goerli is supported for now
+NETWORK_NAME=goerli
+# can be full or archive
+NODE_TYPE=full
+# only download option is available for now
+BEDROCK_SOURCE=download
+# L1 node that the op-node will get chain data from - please use your own ETH node
+OP_NODE__RPC_ENDPOINT=
+# Use basic for OP_NODE__RPC_TYPE if you are using your own node in OP_NODE__RPC_ENDPOINT
+# Other options are: alchemy, quicknode, infura, parity, nethermind, debug_geth, erigon
+# Depending upon which provider you are using in OP_NODE__RPC_ENDPOINT, mentioning the right
+# OP_NODE__RPC_TYPE will optimize the process
+OP_NODE__RPC_TYPE=
+`}
+</CodeBlock>
+}
+]} />
+
+#### Example Variables
+
+<Tabs tabs={[
+{
+title: "Mainnet",
+content: <CodeBlock language="bash">
+{`
+NETWORK_NAME=mainnet
+NODE_TYPE=full
+BEDROCK_SOURCE=download
+OP_NODE__RPC_ENDPOINT=YOUR_MAINNET_ETH_NODE
+OP_NODE__RPC_TYPE=basic
+`}
+</CodeBlock>
+},
+{
+title: "Testnet",
+content: <CodeBlock language="bash">
+{`
+NETWORK_NAME=goerli
+NODE_TYPE=full
+BEDROCK_SOURCE=download
+OP_NODE__RPC_ENDPOINT=YOUR_GOERLI_ETH_NODE
+OP_NODE__RPC_TYPE=basic
+`}
+</CodeBlock>
+}
+]} />
+
+
+## Update variables in the config
+
+```bash
+sed -i "s/NETWORK_NAME *=.*/NETWORK_NAME=$NETWORK_NAME/g" $HOME/simple-optimism-node/.env
+sed -i "s/NODE_TYPE *=.*/NODE_TYPE=$NODE_TYPE/g" $HOME/simple-optimism-node/.env
+sed -i "s/BEDROCK_SOURCE *=.*/BEDROCK_SOURCE=$BEDROCK_SOURCE/g" $HOME/simple-optimism-node/.env
+sed -i "s%OP_NODE__RPC_ENDPOINT *=.*%OP_NODE__RPC_ENDPOINT=$OP_NODE__RPC_ENDPOINT%g" $HOME/simple-optimism-node/.env
+sed -i "s/OP_NODE__RPC_TYPE *=.*/OP_NODE__RPC_TYPE=$OP_NODE__RPC_TYPE/g" $HOME/simple-optimism-node/.env
+```
+
```

Comment:
```suggestion
```
I prefer to point folks to edit the file instead of just replacing so they familiarize themselves with the configs